### PR TITLE
Format the meta content for Slack attachments

### PR DESCRIPTION
**Pull request by alin-o**
**Issues:**
1. The meta content of the message is not displayed in slack
2. When using [mattermost webhooks](https://docs.mattermost.com/developer/webhooks-incoming.html#slack-compatibility) (which are slack compatible), the notifications with meta content are not displayed at all. In the log file, there is an error like this: `IncomingWebhookRequestFromJson code=400 rid=96ru53wc3bg1tytkzs9uiwozfa uid= ip=**** Unable to parse incoming data [details: json: cannot unmarshal string into Go struct field SlackAttachment.fields of type []*model.SlackAttachmentField]`

**Cause:**
the `attachments.fields` is not in the right format. it should be an array of objects with properties `title` and `value` (as documented [here](https://api.slack.com/docs/message-attachments#attachment_structure))

**Solution:**
use a function to translate the meta object in the right format
for easier testing, I also added a meta field to the `TestMessage`

### CODE REVIEW

File: app/Notifications/Drivers/Slack.php [hunk 1]
```diff
@@ -27,12 +27,12 @@ public function fire($recipient, $message)
 
         $body = [
             'text' => $message->title(),
-            'icon_url' => 'https://eyewitness.io/img/logo/icon_192_192.png',
+            'icon_url' => config('logging.channels.slack.emoji_' . ($message->isError() ? 'danger' : 'good') , '/static/emoji/1f441.png'),
```
Comment (laurencei) [inline app/Notifications/Drivers/Slack.php]: What is the purpose of this change? And where is that default image being loaded from?

Comment (alin-o) [reply app/Notifications/Drivers/Slack.php]: sorry, this change should not be in the pull request, I changed it for my own purpose after i made the pull request.

Comment (alin-o) [reply app/Notifications/Drivers/Slack.php]: I made a new pull request instead (https://github.com/eyewitness/eye/pull/46)
